### DISCUSSION TITLE
Has any way to made `paths base` to be case insensitive ?

### DISCUSSION THREAD

**Discussion by kodaicoder** [sveltejs/kit#10965] [Q&A]
I deploy sveltekit project using adaptor-node with paths base configured  in svelte.config.js file , for example I deploy on domain `my.sites.com` with paths base as `base1` then the route to the site should be `my.sites.com/base1` if I access that url with `my.sites.com/base1` it definitely work fine but somehow if I access it with either`my.sites.com/BASE1` or `my.sites.com/Base1` or etc., that not working and showing me a black screen page with "Not found" word (I think it may be a 404.html from sveltekit itself)

Is there some way I can made `paths base` to be case insensitive ?

P.S. I deploy on IIS that using IISNode

Thank you.🙂

**Comment by Angus-Paillaugue** [comment]
I think what you are trying to accomplish is this : [#10207](https://github.com/sveltejs/kit/discussions/10207)
